### ISSUE TITLE
[8.0] api xml: newlines in groups tags ignored

### ISSUE THREAD

**Issue by moylop260** [odoo/odoo#9797]
Steps:
Add a xml with newline in `groups` tag
```xml
..."groups">group1,\n
group2</..
```

Saw:
Just the group1 was assigned.
The group2 was ignored.

Expected:
That show a error of newline don't supported or that newline works

**Comment by moylop260** [comment]
@xmo-odoo 
Do you have news?

**Comment by xmo-odoo** [comment]
Hadn't had the occasion to really look at it. Now that I do, the overly short code snippet is odd, would it be possible to have more context? Is this an `<attribute>` tag or something else?

**Comment by moylop260** [comment]
@xmo-odoo 
Thanks for reply:

```diff
             </xpath>
             <xpath expr="//page[@string='Extra Info']" position="attributes">
-                <attribute name="groups">
-                    yoytec_customer_rma_workflow.group_rma_technical,
-                    yoytec_customer_rma_workflow.group_rma_leader</attribute>
+                <attribute name="groups">yoytec_customer_rma_workflow.group_rma_technical,yoytec_customer_rma_workflow.group_rma_leader</attribute>
             </xpath>
             <xpath expr="//field[@name='repaired']" position="after">
                 <field name="diagnosed"/>
```

**Comment by xmo-odoo** [comment]
Should be fixed by 7c0e734

**Comment by moylop260** [comment]
@xmo-odoo 
I got it
Thank you!
